### PR TITLE
Added Pd::Enrollment.skip_school_validation

### PR DESCRIPTION
**Pull request by aoby**
... to allow legacy enrollments before those fields were required

Yesterday we made school and school_info required, and then old enrollments failed [update](https://github.com/code-dot-org/code-dot-org/blob/7fd5633d15265f217787728ef25e2dea609e8a65/dashboard/app/mailers/pd/workshop_mailer.rb#L96) while sending exit survey emails.

### CODE REVIEW

File: dashboard/app/mailers/pd/workshop_mailer.rb [hunk 1]
```diff
@@ -93,7 +93,8 @@ def exit_survey(workshop, teacher, enrollment)
       CDO.warn "Skipping attempt to send a duplicate workshop survey email. Enrollment: #{enrollment.id}"
       return
     end
-    enrollment.update!(survey_sent_at: Time.zone.now)
+    # Skip school validation to allow legacy enrollments before those fields were required to update.
```
Comment (breville) [inline dashboard/app/mailers/pd/workshop_mailer.rb]: suggest: "Skip school validation to allow legacy enrollments (from before those fields were required) to update."

**Comment by breville** [comment]
lgtm.  nice testing.

**Comment by aoby** [comment]
The remaining test failures are unrelated UI tests. All unit tests are passing (circle node 0). Merging.

**Comment by aoby** [comment]
The circleCI failures are all unrelated UI test connection issues. This PR didn't modify any UI, and all the unit tests passed. Merging...

**Pull request merged**